### DISCUSSION TITLE
Can we log background traffic?

### DISCUSSION THREAD

**Discussion by ahmetax** [ItzCrazyKns/Vane#405] [General]
It would be very nice to see the details of background traffic of Perplexica using a logging mechanism. 
For example,now, all parts of my Perplexica docker system (ollama, searxng) run separately with no problems. However some prompts run forever with no answer. 
At least, maybe a timeout parameter should be added.
If the result is not successful, maybe a message can be added to the library after a timeout period.

**Comment by ngdbao** [comment]
if using Cloud AI api, just edit endpoint to proxied via Cloudflare gateway AI, with full log
